### PR TITLE
Add 'online' column for Country

### PR DESCRIPTION
**Pull request by Si-plus-plus**
- Add `online` column with Boolean data type for Country table
- Add prefilled `online` column for countries2019.csv

### CODE REVIEW

**Review by fushar — APPROVED** [review]

File: data/countries2019.csv [hunk 1]
```diff
@@ -1,88 +1,89 @@
...
+Tunisia,TUN,TN,0
+Turkey,TUR,TR,0
+Taiwan,TWN,TW,0
+Ukraine,UKR,UA,0
+United States of America,USA,US,0
+Uzbekistan,UZB,UZ,0
+Venezuela,VEN,VE,0
+Vietnam,VNM,VN,0
+South Africa,ZAF,ZA,0
+,0
```
Comment (fushar) [inline data/countries2019.csv:89]: Looks like this is not intended

**Review by fushar — CHANGES_REQUESTED** [review]

**Review by raisfathin — CHANGES_REQUESTED** [review]
I think there should be a migration file in trans/migrations. That needs to be committed as well. Did `manage.py makemigrations` not generate any?

File: data/countries2019.csv (hunk 1 above)
Comment (Si-plus-plus) [reply data/countries2019.csv]: missed that, fixed.

**Comment by Si-plus-plus** [comment]
> I think there should be a migration file in trans/migrations. That needs to be committed as well. Did `manage.py makemigrations` not generate any?

image

It should have generated one, but it didn't. The bottommost file inside the trans/migrations folder was created last year, while the rest was from 2017-2018.

**Comment by fushar** [comment]
> It should have generated one, but it didn't. 

I just tried, it did generate a new migration file:

```
root@08b05b9ed9ed:/usr/src/app# python3 manage.py makemigrations
(0.003)
            SELECT c.relname, c.relkind
            FROM pg_catalog.pg_class c
            LEFT JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace
            WHERE c.relkind IN ('r', 'v')
                AND n.nspname NOT IN ('pg_catalog', 'pg_toast')
                AND pg_catalog.pg_table_is_visible(c.oid); args=None
(0.001) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
Migrations for 'trans':
  trans/migrations/0018_country_online.py
    - Add field online to country
```

The file was created inside the docker container, so you need to scp it out, or just copy-paste the content :)

**Comment by Si-plus-plus** [comment]
> > It should have generated one, but it didn't.
> 
> I just tried, it did generate a new migration file:
> 
> ```
> root@08b05b9ed9ed:/usr/src/app# python3 manage.py makemigrations
> (0.003)
>             SELECT c.relname, c.relkind
>             FROM pg_catalog.pg_class c
>             LEFT JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace
>             WHERE c.relkind IN ('r', 'v')
>                 AND n.nspname NOT IN ('pg_catalog', 'pg_toast')
>                 AND pg_catalog.pg_table_is_visible(c.oid); args=None
> (0.001) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
> Migrations for 'trans':
>   trans/migrations/0018_country_online.py
>     - Add field online to country
> ```
> 
> The file was created inside the docker container, so you need to scp it out, or just copy-paste the content :)

Thanks for pointing it out. I wouldn't have realized it otherwise

**Review by raisfathin — APPROVED** [review]

**Review by fushar — APPROVED** [review]

**Pull request merged**